server/chunker.js

```diff
@@ -152,7 +152,7 @@ const getSemanticChunks = (text, chunkSize, inputFormat = 'text') => {
       }
     });
 
-    if (chunks.some(chunk => encode(chunk).length > chunkSize)) {
+    if (chunks.filter(c => determineTextFormat(c) === 'html').some(chunk => encode(chunk).length > chunkSize)) {
```

Comment:
Do we have HTML chunking tests for this?

Reply:
We do have a test that checks if the chunking function will chunk a long blob of text between two HTML elements:

https://github.com/aj-archipelago/cortex/blob/9d9a535c8fa18620134893da9938d61b45e15846/tests/chunkfunction.test.js#L85C14-L85C26